rules-tests/CodeQuality/Rector/Class_/PreferPHPUnitSelfCallRector/Fixture/include_exceptions.php.inc

```diff
@@ -8,6 +8,7 @@ final class IncludeExceptions extends TestCase
 {
```

Comment:
You can rename fixture with `skip_include_exceptions.php.inc`, then remove content start from `-----` as equal

Reply:
Then, add fixture on `PreferPHPUnitThisCallRector` to change `self::expectException()` to `$this->expectException()`

Reply:
> You can rename fixture with skip_include_exceptions.php.inc, then remove content start from ----- as equal

done

> Then, add fixture on PreferPHPUnitThisCallRector to change self::expectException() to $this->expectException()

It already exists `rules-tests/CodeQuality/Rector/Class_/PreferPHPUnitThisCallRector/Fixture/expect_exception.php.inc`